assets/src/edit-story/components/library/libraryProvider.js

```diff
@@ -22,30 +22,37 @@ const LINKS = 'links';
 function LibraryProvider( { children } ) {
 	const { actions: { getMedia } } = useAPI();
 	const [ media, setMedia ] = useState( [] );
+	const [ perPage, setPerPage ] = useState( 100 );
```

Comment:
Will it be possible to set `perPage` by the user?

Reply:
This was added for future use.

Reply:
I can't see the option in Figma for changing this at all, could we for now just define a constant for that and only add if it should be needed? Also, for the purposes of YAGNI the architecture is following.

Reply:
This is more useful to the lazy loading to receive more assets when scrolling to the end of the media library.

Reply:
Decided you are right here, we can fix this later. Removed in https://github.com/ampproject/amp-wp/pull/3843/commits/0bc9410aabf0a28fe3df06f1f4969cb49710869e